packages/services/api/src/modules/target/module.graphql.ts

```diff
@@ -178,11 +181,35 @@ export default gql`
   type TargetValidationSettings {
     enabled: Boolean!
     period: Int!
+
+    """
+    If TargetValidationSettings.breakingChangeFormula is PERCENTAGE, then this
+    is the percent of the total operations over the TargetValidationSettings.period
+    required for a change to be considered breaking.
+    """
     percentage: Float!
```

Comment:
Should this be `Float` now? 🤔 (without `!`) 
I mean, in case it's being queried and the user is using `REQUEST_COUNT`, this should be empty? 

same for `requestCount: Int!`?

Reply:
We have a default value and both values are stored in the database, so having them non-nullable is fine.

Reply:
I wanted to avoid making a breaking change to the schema